sites/example-project/src/components/modules/getColumnSummary.js

```diff
@@ -42,6 +46,7 @@ export default function getColumnSummary(data, returnType="object") {
         colName = key;
         colFmtTag = getFormatTag(key);
         colType = getColumnType(data, colName, colFmtTag);
+        evidenceColumnType = getColumnEvidenceType(data, colName);
```

Comment:
Why can't we just accept the result of `getColumnEvidenceType` as the colType that we pass on to components relying on this?

Reply:
Or is that just the next iteration of this?